extensions/amp-ad/amp-ad.md

```diff
@@ -1,5 +1,7 @@
 # <a name="amp-ad"></a> `amp-ad` / `amp-embed`
 
+[TOC]
```

Comment:
hmmm there's still conflict here, due to previous incorrect merging I think. Can you double check and fix. You should be able to tell from git file changed diff directly. There're should be only one line add to the README file.